xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/LikeUIX.xml

```diff
@@ -321,7 +345,54 @@ require(['jquery', 'xwiki-meta', 'xwiki-l10n!xwiki-like-messages'], function ($,
 }
 .like-container {
   /* Leave some space between the like button and the next footer item (e.g. the page tags). */
-  margin-right: 2em;
+  padding: 3px 2em 3px 0;
+
+  .like-button {
+    background: transparent;
+    border: none;
+    box-shadow: none;
+    border-radius: @btn-border-radius-small;
+    font-size: 1.33em;
+    line-height: 1;
+    vertical-align: middle;
+    padding: 0;
+    margin: 0;
+    outline: none;
+
+    &amp;:active, &amp;:focus, &amp;:active:focus {
+      outline: none;
+    }
```

Comment:
I think this is also an accessibility issue.

Reply:
It's not advised for accessibility because by itself it breaks [this WCAG success criterion](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html) :+1: 
See [this article from the a11yproject](https://www.a11yproject.com/posts/never-remove-css-outlines/) and [this one on WebAIM](https://webaim.org/blog/plague-of-outline-0/) for more info.
Note that we already have about 10 occurences of this on xwiki-platform: https://github.com/search?q=repo%3Axwiki%2Fxwiki-platform%20outline%3A%20none&type=code
Some might be correct uses (something different than the outline is changed on focus) but I doubt all of them are :)
I'll propose a css codestyle rule about this and open a ticket to track and fix those uses.